cache-config/testing/ort-tests/tcdata/todb.go

```diff
@@ -101,7 +101,7 @@ func SetupRoles(db *sql.DB) error {
 	sqlStmt := `
 INSERT INTO role (name, description, priv_level) VALUES ('disallowed','Block all access',0) ON CONFLICT DO NOTHING;
 INSERT INTO role (name, description, priv_level) VALUES ('read-only','Block all access', 10) ON CONFLICT DO NOTHING;
-INSERT INTO role (name, description, priv_level) VALUES ('operations','Block all access', 20) ON CONFLICT DO NOTHING;
+INSERT INTO role (name, description, priv_level) VALUES ('operations','Operations-user', 30) ON CONFLICT DO NOTHING;
```

Comment:
The "operations" Role as created by `seeds.sql` on new ATC/TO installs has privilege level 20. If you need admin-level Permissions, why not just use the admin Role instead of confusing this concept?